CHANGELOG.md

```diff
@@ -18,7 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Implement `MethodError` for `safe_erc20::Error`. #402
 - Use `function_selector!` to calculate transfer type selector in `Erc1155`. #417
-- Update internal functions of `Erc721` and `Erc721Consecutive` accept reference to `Bytes`. #437 
+- Update internal functions of `Erc721` and `Erc721Consecutive` to accept a reference to `Bytes`. #437
+- Replace all `Call:new_in` to `Call:new` to stop supporting reentrancy. #440
```

Comment:
I would say it should be marked as `### Changed (Breaking)`

Reply:
Thanks for the review @0xNeshi @bidzyyys ! Added `### Changed (Breaking)` section